### PR TITLE
10. GET /api/users

### PR DESCRIPTION
**Pull request by dafyddhenke**
Card id: https://trello.com/c/86J6ALal

- User can request users endpoint
- Returns users array with username, name, and avatar_url
- Tests that valid response

### CODE REVIEW

File: controllers/users.controllers.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const { selectUsers } = require("../models/users.models");
+
+exports.getUsers = (req, res, next) => {
+  selectUsers()
+    .then((returnedUsers) => {
+      res.status(200).send(returnedUsers);
```
Comment (two-triangles) [inline controllers/users.controllers.js:6]: we should be sending back our user in an object, with a key that we can then access in the response body - {users: returnedUsers}

Comment (dafyddhenke) [reply controllers/users.controllers.js:6]: Sure, can do that.

Comment (dafyddhenke) [reply controllers/users.controllers.js]: Done.

File: __tests__/users.controller.test.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      .then((response) => {
+        expect(Array.isArray(response.body)).toBe(true);
+      });
+  });
+  test("GET /api/users should return an an array of objects with username, name, and avatar_url props", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then((response) => {
+        const usersArray = response.body;
```
[the remark this answers is not in the document]
Comment (dafyddhenke) [reply __tests__/users.controller.test.js]: Done.

**Pull request merged**